README.md

```diff
@@ -1,36 +1,134 @@
+`To` tags. An example is available below.
+```xml
+<h:AppHdr xmlns:h="urn:iso:std:iso:20022:tech:xsd:head.001.001.02">
+    <h:Fr>
+        <h:FIId>
+            <h:FinInstnId>
+                <!--The sending financial institution has the hostname "example.org" and its external API runs on port 443-->
+                <h:Nm>example.org</h:Nm>
+            </h:FinInstnId>
+        </h:FIId>
+    </h:Fr>
+    <h:To>
+        <h:FIId>
+            <h:FinInstnId>
+                <!--The receiving financial institution has the hostname "example.com" and its external API runs on port 1234-->
+                <h:Nm>example.com:1234</h:Nm>
+            </h:FinInstnId>
+        </h:FIId>
+    </h:To>
+    <!--Further tags here...-->
+</h:AppHdr>
+<!--ISO 20022 message here...-->
+```
+The `Fr` tag must contain the hostname of your financial institution. Specifically, it must be a subject alternative name of your
+personal X.509 certificate. The `To` tag will be used by the microservice to determine the financial institution the message should
+be sent to. The port numbers of both the sending and receiving financial institution's external APIs can optionally be specified, but
+if they are not present they are assumed to be located on port 443.
+
+After the message is constructed, it should be sent to your financial institutions internal API by sending a HTTP request using the
+POST method to its `/api/v1/pacs` endpoint. The response can either be a `pacs.002` message indicating success, or an error message.
+An example of an error message can be found below. For a list of all error statuses that can be returned by the internal and
+external APIs, see the [OpenAPI description](OpenAPI/decentralized-iso20022-v1.yaml)
+
+```xml
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>                                                            <!--The XML header-->
+<error>
+    <timestamp>2021/04/21 10:36:58</timestamp>                                                                    <!--The time that the error occurred-->
+    <status>502</status>                                                                                          <!--The HTTP status of the error message-->
+    <code>BAD GATEWAY</code>                                                                                      <!--The Reason phrase of the HTTP status-->
+    <message>The remote financial institution failed to handle the message. Please try again later.</message>     <!--A detailed error message-->
+</error>
+```
+
+### 6.4 Receiving Messages
+The external API which receives messages from other financial institutions will only validate the messages it receives, it will not "act"
+upon them. Therefore, after it has validated a message, it will expect that a "financial institution handler" will act upon it.
+The URI of this handler must be specified in `application-external.properties` in accordance with [6.4. Configuration](#64-configuration).
```

Comment:
Is this maybe supposed to link to 6.5? and not 6.4 which is itself?

Reply:
Well spotted, fixed!